src/features/home/components/account-overview-ribbon.tsx

```diff
@@ -15,63 +16,77 @@ const AccountOverviewRibbon = () => {
     data: investorStats,
     isPending: statsPending,
     isError: statsError,
-  } = useQuery(investorsMeStatsRetrieveOptions());
+  } = useQuery(statisticsStatsRetrieveOptions());
 
   const {
     data: currentAccountValue,
     isPending: accountValuePending,
     isError: accountValueError,
-  } = useQuery(investorsMeCurrentAccountValueRetrieveOptions());
+  } = useQuery(statisticsCurrentAccountValueRetrieveOptions());
```

Comment:
magically, when the account value and owned money is 0 - so creating an account or equivalent, the error is displayed here, upon depositing money in the wallet, the error disappears
<img width="755" height="168" alt="image" src="https://github.com/user-attachments/assets/1ba4b13d-0206-4107-9a6d-60bd65d7ecfb" />